### PR TITLE
fix(hosting.logs): use API log url instead of constant

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/web-ca`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-5056
| License          | BSD 3-Clause

## Description

use API for logs for all logs constants

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#12239.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#12239.1 ✘  
* 
  * Build ✘

File: packages/manager/apps/web/client/app/hosting/user-logs/USER_LOGS.html [hunk 1]
```diff
@@ -20,54 +20,46 @@ <h2 data-translate="hosting_tab_USER_LOGS"></h2>
...
+                </div>
+                <div data-ng-if="ctrl.logs.logs">
+                    <dt
+                        data-translate="hosting_tab_USER_LOGS_dashboard_logs"
+                    ></dt>
+                    <dd>
+                        <a
+                            class="oui-link_icon"
+                            data-ng-href="{{ctrl.userLogsToken ? ctrl.logs.logs + '?token=' +  ctrl.userLogsToken : ctrl.logs.logs }}"
+                            target="_blank"
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/user-logs/USER_LOGS.html:52]:
```suggestion
                            target="_blank"
                            rel="noopener"
```

**Comment by marie-j** [comment]
Added suggestion in https://github.com/ovh/manager/pull/2973#discussion_r426685631

**Pull request merged**